DigitalLearningSolutions.Web/Views/LearningPortal/Current/MarkActionPlanItemAsComplete.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@using Microsoft.Extensions.Configuration
+@model SetLearningLogItemCompletionDateViewModel
+@inject IConfiguration Configuration
+
+@{
+  ViewData["Title"] = $"Mark {Model.ResourceName} as complete";
+  ViewData["Application"] = "Learning Portal";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningPortal/Current";
+  ViewData["HeaderPathName"] = "My Current Activities";
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  var exampleDate = DateTime.Today;
+}
+
+@section NavMenuItems {
+  <partial name="Shared/_NavMenuItems" />
+}

```

Comment:
Can replace most of this by using SetDlsSubApplication attribute on the new action.